packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx

```diff
@@ -58,10 +58,13 @@ const Success: React.FC<Props> = props => {
         <FlyoutWrapper>
           <TopText color='grey800' size='20px' weight={600}>
             <span>
-              <BuyOrSell orderType={getOrderType(props.order)} />{' '}
-              <FormattedMessage
-                id='modals.simplebuy.order_summary'
-                defaultMessage='Order Summary'
+              <BuyOrSell
+                orderType={orderType}
+                crypto={
+                  orderType === 'BUY'
+                    ? (props.order.outputCurrency as CoinType)
+                    : (props.order.inputCurrency as CoinType)
```

Comment:
Can you use `getCoinFromPair(props.order.pair)` instead?

Reply:
Done.